sysutils/kubescape/Portfile

```
@@ -17,25 +17,18 @@ license             Apache-2
 maintainers         {gmail.com:herby.gillot @herbygillot} \
                     openmaintainer
 
-checksums           rmd160  c64707196896f61b0d521a304f4e85850a0605e5 \
-                    sha256  b0353cc3fb2f56a9370e9cdd13ddbf0a32bec8f0613652a0037e82556e37229c \
-                    size    54776060
+checksums           rmd160  55cef26067bdbd9fe6c5531649a3e3df6ecdc481 \
+                    sha256  6430de50ffecd5396273dd00941f995ace76533cfe5937630c716e27209dd4be \
+                    size    9748553
 
 # Allow Go to fetch dependencies at build time
 go.offline_build no
-build.env-append    RELEASE=${version}
-
-set py_ver          310
-set py_branch       [string index ${py_ver} 0].[string range ${py_ver} 1 end]
-
-depends_build-append \
-                    port:python${py_ver}
-
-build.cmd           ${prefix}/bin/python${py_branch}
-build.target        ./build.py
+build.pre_args-append -ldflags \" \
+    -X ${go.package}/v3/core/cautils.BuildNumber=v${version} \
+    -X ${go.package}/v3/core/cautils.Client=release \
+    \"
+build.cmd           "go mod tidy && go build"
```

One last nitpick, let's have the Portfile reference Go using the variable provided by the `golang` portgroup instead of hardcoding it. This is a change that actually needs to be made in a more widespread fashion across Portfiles for various Go ports.

```suggestion
build.cmd           "${go.bin} mod tidy && ${go.bin} build"
```